modules/setting/indexer.go

```diff
@@ -40,6 +41,7 @@ var Indexer = struct {
 	IssueIndexerName: "gitea_issues",
 
 	RepoIndexerEnabled: false,
+	RepoIndexerUnits:   []string{"repo", "fork", "mirror"},
```

Comment:
And here is `	RepoIndexerUnits:   []string{"repo", "fork", "mirror"},`

Three kinds of concepts?